docs/pipeline-standards.md

```diff
@@ -15,241 +15,244 @@ Libuit KG, Guthrie J, Ambrosio F, Kapsak C, Unal Gultekin, Holmes J, Wright S, N
 
-A clear commitment to maintaining software for public health bioinformatics ensures long-term stability, reliability, and functionality, ultimately enhancing the efficacy of research and analysis. This commitment should be made for a specific duration, with a maximum limit of two years. This timeframe strikes a balance between providing sufficient longevity for user confidence and preventing the commitment from becoming stale or outdated. By stipulating a maximum duration, it encourages regular reassessment and renewal of the commitment, ensuring that the software remains relevant and up-to-date. Moreover, it fosters user confidence and facilitates wider adoption, as researchers can rely on consistent updates, improvements, and support for the software within this defined period.
+Containers are essential for modern bioinformatics development and pipeline distribution, as the ability to replicate results is a fundamental principle of the scientific method. This is true for public health, as any lab should be able to easily install and maintain pipeline and reproduce/verify results from another lab. Containers should be packaged with one or a combination of the following methods:
+- conda environments
+- venv environments
+- Singularity containers
+- Docker containers.
 
-- To meet this standard, clearly communicate a commitment to maintaining the bioinformatics software for public health, indicating a dedication to stability, reliability, and functionality for at leat 2 years.
-- To verify this standard, the reviewer should look for explicit statements or documentation expressing a commitment to ongoing maintenance, with evidence of regular updates, improvements, user support, and easily accessibly means of contact. Bi-annual re-verification will require the reviewer to reach out to the point of contact, and update if necessary.
+There should be a clear summary in the Git README pointing to which containerisation method has been chosen and instructions for how a lab can install this pipeline and/or where docker images are available, e.g. dockerhub and quay for containers; and where conda packages are available, e.g. anaconda and bioconda (cross-referenced with Installation Instructions).
 
-### Conflict of Interest Statement
-_Have all potential conflicts of interest been disclosed?_
+When bioinformatics software is containerised, there should be a supplementary document justifying, where appropriate, why a particular version was chosen for analysis. Reasons for choosing specific pipeline versions include but are not limited to ensuring functionality by selecting compatible dependencies and major differences in functionality and features between pipeline versions.
```

Comment:
Maybe it is not necessary to justify a specific version if it was the latest version when the pipeline release was done?

Reply:
I think it's not clear here if you are speaking about software version or pipeline version. I'm a little bit confused.

Reply:
Ah, thanks for catching this. Updating the language to clarify :

`There should be a clear summary in the Git README pointing to which containerisation method has been chosen and instructions for how a lab can install this pipeline and/or where docker images are available, e.g. dockerhub and quay for containers; and where conda packages are available, e.g. anaconda and bioconda (cross-referenced with Installation Instructions). Documentation should indicate the specific version included in the pipeline. This is important as specific software versions  may impact functionality.`